_opensearch/rest-api/rank-eval.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Ranking evaluation
+parent: REST API reference
+nav_order: 82
+---
+
+# Ranking evaluation
+
+The [rank]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/rank/) eval endpoint allows you to evaluate the quality of ranked search results.
+
+## Path and HTTP methods
+
+```
+GET <index_name>/_rank_eval 
+POST <index_name>/_rank_eval
+```
+
+## URL parameters
+
+URL Parameters are optional.
+
+Parameter | Data Type | Description
+:--- | :---  | :---
+[`ignore_unavailable`](#ignore_unavailable) | boolean | Defaults to `false`. When set to `false` the response body will return an error if an index is closed or missing.
```

Comment:
Are these links going to be in the same file?

Reply:
yeah, I was on the fence with this as I was formatting so I think I'll just remove these are most other pages don't match up